src/braket/timings/time_series.py

```diff
@@ -186,45 +185,18 @@ def stitch(
 
         Args:
             other (TimeSeries): The second time series to be stitched with.
-            boundary (StitchBoundaryCondition): `{"mean", "left", "right"}`. Boundary point handler.
-              Possible options are
+            boundary (StitchBoundaryCondition): {"mean", "left", "right"}. Boundary point handler.
+
+                Possible options are
 
-                - "mean" - take the average of the boundary value points of the first
-                  and the second time series.
-                - "left" - use the last value from the left time series as the boundary point.
-                - "right" - use the first value from the right time series as the boundary
-                  point.
+              - "mean" - take the average of the boundary value points of the first
```

Comment:
I think you need to indent this by 4 if you want it to look right.

Reply:
If we indent by 4, half the text is rendered as bold.